src/content/post/2025-04-15-embedded-dbs.md

```diff
@@ -116,25 +112,24 @@ Aside from the above two advantages, in-process DBMSs can also have some perform
    directly access and scan the contents of the data frame 
    without incurring any real data movement costs, e.g., any serialization/deserialization or data copies[^1].
 
-##  Does in-process mean "ephemeral databases"?
+##  Does in-process mean "ephemeral"?
 
-Let me next discuss some common misconceptions about in-process DBMSs. 
-Because in-process DBMSs are libraries and popular data science libraries, such as Pandas and Polars,
-do not persist your data, a common question I hear is: "Does in-process DBMS mean the DBMS only supports ephemeral databases?".
-The answer is no! Just like any other DBMS, all in-process DBMSs I mentioned above persist your databases on disk.
-However, many of the in-process DBMSs, e.g., SQLite, DuckDB, and Kuzu have an "in-memory" mode[^2] if you do not want to persist your data.
-For example, in Kuzu, if you create a `Database` object with an empty string, e.g., `db = kuzu.Database("")`,
-you will create an ephemeral database that is not persisted on disk. 
+I'll next address a common misconception about in-process DBMSs. 
+A question I often get is: “Since in-process DBMSs are libraries—and popular data science libraries like Pandas and Polars don’t persist data—does that mean in-process DBMSs only support ephemeral databases?”
```

Comment:
Let's change the 2nd sentence to: "Because in-process DBMSs are Pandas-like data science libraries and because these libraries do non persist you data, a question I often get is: "Are the databases I create using in-process DMBSs also ephemeral?"